tests/flow/test_procedures.py

```diff
@@ -133,137 +133,144 @@ def test_arguments(self):
                     LIMIT 1
                     """
         expected_results = [node2]
-        self.queryAndValidate(query, expected_results)
+        self.queryAndValidate(query, expected_results, query_params=query_params)
 
 
         # CALL + WHERE + RETURN + ORDER.
-        query = """CALL db.idx.fulltext.queryNodes('fruit', 'Orange*') YIELD node
+        query = """CALL db.idx.fulltext.queryNodes('fruit', $prefix) YIELD node
                     WHERE node.value > 2
                     RETURN node
                     ORDER BY node.value"""
         expected_results = [node3, node4]
-        self.queryAndValidate(query, expected_results)
+        self.queryAndValidate(query, expected_results, query_params=query_params)
 
 
         # CALL + WHERE + RETURN + ORDER + SKIP.
-        query = """CALL db.idx.fulltext.queryNodes('fruit', 'Orange*') YIELD node
+        query = """CALL db.idx.fulltext.queryNodes('fruit', $prefix) YIELD node
                     WHERE node.value > 2
                     RETURN node
                     ORDER BY node.value
                     SKIP 1"""
         expected_results = [node4]
-        self.queryAndValidate(query, expected_results)
+        self.queryAndValidate(query, expected_results, query_params=query_params)
 
 
         # CALL + WHERE + RETURN + ORDER + LIMIT.
-        query = """CALL db.idx.fulltext.queryNodes('fruit', 'Orange*') YIELD node
+        query = """CALL db.idx.fulltext.queryNodes('fruit', $prefix) YIELD node
                     WHERE node.value > 2
                     RETURN node
                     ORDER BY node.value
                     LIMIT 1"""
         expected_results = [node3]
-        self.queryAndValidate(query, expected_results)
+        self.queryAndValidate(query, expected_results, query_params=query_params)
 
 
         # CALL + WHERE + RETURN + ORDER + SKIP + LIMIT.
-        query = """CALL db.idx.fulltext.queryNodes('fruit', 'Orange*') YIELD node
+        query = """CALL db.idx.fulltext.queryNodes('fruit', $prefix) YIELD node
                     WHERE node.value > 2
                     RETURN node
                     ORDER BY node.value
                     SKIP 1
                     LIMIT 1"""
         expected_results = [node4]
-        self.queryAndValidate(query, expected_results)
+        self.queryAndValidate(query, expected_results, query_params=query_params)
 
         # CALL + MATCH + RETURN.
-        query = """CALL db.idx.fulltext.queryNodes('fruit', 'Orange*') YIELD node
+        query = """CALL db.idx.fulltext.queryNodes('fruit', $prefix) YIELD node
             MATCH (node)-[]->(z)
             RETURN z"""
         expected_results = [node5]
-        self.queryAndValidate(query, expected_results)
+        self.queryAndValidate(query, expected_results, query_params=query_params)
+
+        # UNWIND + CALL + RETURN.
+        query = """UNWIND([1,2]) AS x CALL db.idx.fulltext.queryNodes('fruit', $prefix) YIELD node RETURN node"""
```

Comment:
This query _should_ emit a syntax error because it has `UNWIND([1,2])` instead of `UNWIND [1,2]`, and apparently doesn't because parameters are passed? I'm not really sure what to make of that, but it should be looked into here or elsewhere!

Reply:
I believe UNWIND operates on an expression which is evaluated to an array, in which case the parentheses has no effect, and so I don't see an issue with that.

Reply:
:+1: You're right, my bad!